lib/rad_common/engine.rb

```diff
@@ -40,5 +40,6 @@ class Engine < ::Rails::Engine
     require 'turbo-rails'
     require 'twilio-ruby'
     require 'wicked_pdf'
+    require 'tty-prompt'
```

Comment:
if it's only for development, I think this can be moved into the Gemfile in rad_common